builtin/update-microsoft-git.c

```diff
@@ -0,0 +1,70 @@
+#include "builtin.h"
+#include "repository.h"
+#include "parse-options.h"
+#include "run-command.h"
+#include "strvec.h"
+
+#if defined(GIT_WINDOWS_NATIVE)
+/*
+ * On Windows, run 'git update-git-for-windows' which
+ * is installed by the installer, based on the script
+ * in git-for-windows/build-extra.
```

Comment:
Does the microsoft/git version of build-extra point `upgrade-git-for-windows` to a different URL than a normal GFW installation ??

Maybe link to that.

Reply:
[Here is the script](https://github.com/git-for-windows/build-extra/blob/HEAD/git-extra/git-update-git-for-windows) but the real change is inside our Azure Pipeline build (yikes). See #321 for details.